.gitattributes

```diff
@@ -0,0 +1 @@
+./client/app/visualizations/choropleth/maps/australia.postcodes.geo.json filter=lfs diff=lfs merge=lfs -text
```

Comment:
What's this about?

Not sure if related, but this PR is missing the actual map JSON files.

Reply:
yeah I tried to upload it, but its hitting the limits of github file sizes. 
 I will down-sample the map and create another PR for this.

Reply:
@kravets-levko is working on a way to load custom maps without adding them to the repo (#4599), so maybe just wait for it and load the map from an external source?